### ISSUE TITLE
Presets documentation

### ISSUE THREAD

**Issue by jlaffaye** [zaidka/genieacs#115]
The presets documentation on the wiki page is empty (preconditions, ...)

**Comment by zaidka** [comment]
This is probably one of the most critical bugs submitted. Not sure when I'll be able to dedicate some time for this, but I'm available through email or IM for anyone willing to contribute to the wiki.

**Comment by DrumSergio** [comment]
Yes, we will have to document it. But better to put it in the genieacs-gui project, isn't it? Now there's some info but very simple https://github.com/zaidka/genieacs-gui/wiki/Tab---Presets

**Comment by ujeisharon** [comment]
Cloud there be any document provide more info than https://github.com/zaidka/genieacs-gui/wiki/Tab---Presets ? Ex: How to use the Channnel and Weight ? And when I use the events with "1 BOOT" or "1 BOOT, 2 PERIODIC"  and  configuration never triggered.  It only works with "2 PERIODIC". Is it normal?
Thanks.

**Comment by akcoder** [comment]
GitHub is for defect tracking, not support requests. Please close this ticket and ask your question on the mailing list. 

-dan
